packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -69,21 +69,23 @@ export const ClayBreadcrumbNav: React.FunctionComponent<IProps> = ({
 		: items;
 
 	return (
-		<ol {...otherProps} className={classNames('breadcrumb', className)}>
-			{breadCrumbItems.map(
-				(item: IBreadcrumbItem | React.ReactNode, i: number) =>
-					React.isValidElement(item) ? (
-						React.cloneElement(item, {key: `ellipsis${i}`})
-					) : (
-						<BreadcrumbItem
-							active={(item as IBreadcrumbItem).active}
-							href={(item as IBreadcrumbItem).href}
-							key={`breadcrumbItem${i}`}
-							label={(item as IBreadcrumbItem).label}
-							onClick={(item as IBreadcrumbItem).onClick}
-						/>
-					)
-			)}
-		</ol>
+		<nav aria-label="Breadcrumb">
```

Comment:
I'm wondering if wrapping this in a `nav` can prevent it from composing with other components. And if you follow that, the `otherProps` and `className` should be in `nav`. I'm not sure about that but I prefer to keep it as it is for now because it doesn't stop people from adding...